docs/pages/deploying-vclusters/high-availability.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: High Availability
+sidebar_label: High Availability
+---
+
+import HighAvailabilityK3s from '../fragments/high-availability-k3s.mdx';
+import HighAvailabilityK8s from '../fragments/high-availability-k8s.mdx';
+
+vcluster supports high-availability with k3s as well as the vanilla k8s distribution. k0s is currently not supported for high availability setup of vcluster.
+
+## k3s
+
+<HighAvailabilityK3s />
+
+## Vanilla k8s
+
+<HighAvailabilityK8s />
```

Comment:
@neogopher perhaps moving this from fragments to inline should give us better sidebar nested menus ref image below

![image](https://github.com/loft-sh/vcluster/assets/7482025/c6c9d7c2-47fb-43f9-b753-0482cfc0a35c)

vs 

![image](https://github.com/loft-sh/vcluster/assets/7482025/14f4bbf4-d071-4c28-87ab-3f0488d456ac)


The `### Headings` don't get rendered in right sidebar as subheadings when imported from fragments it seems?

Let me know what you think.